lib/renderer/api/module-list.js

```diff
@@ -1,6 +1,12 @@
+const features = process.atomBinding('features')
+
 // Renderer side modules, please sort alphabetically.
 module.exports = [
-  {name: 'desktopCapturer', file: 'desktop-capturer'},
+  {
+    name: 'desktopCapturer',
+    file: 'desktop-capturer',
+    enabled: features.isDesktopCapturerEnabled()
```

Comment:
_(minor)_ since the default of a boolean is false by convention, it would be slightly better to use something like `.disabled` rather than `.enabled` to imply that, if the property is missing, the module is not disabled

Reply:
I'm strongly against

Reply:
I agree that there is a code smell here with the double negative. I'd argue that we're trading one smell for another wrt the three cases:

  * enabled property exists and is true, means enabled
  * enabled property exists and is false, means disabled
  * enabled property doesn't exist, means enabled

...which is also weird.

A cleaner way would be to have the `.enabled` property everywhere and to not use double negatives

Reply:
> - enabled property exists and is true, means enabled
> - enabled property exists and is false, means disabled
> - enabled property doesn't exist, means enabled
> 
> ...which is also weird.

What exactly do you find weird here? The last clause?

Reply:
Would it be better if the code processing this list were written like this?:
```js
for (const {name, file, enabled = true, private = false} of moduleList) {
  if (!enabled) {
    continue
  }
  <...>
}
```

Reply:
@ckerr 
I don't mind adding `enabled: true` properties though =)

Reply:
@ckerr Which option do you like best?

Reply:
@ckerr I just added a new config parsing code here
https://github.com/electron/electron/pull/13133/files#diff-a2cb68582030a995142a60862e3c1d51

and a comment to this file.

Reply:
To me, the option that would be the least surprising when reading the code for the first time would be if there was an .enabled property for all items, so that we wouldn't have to wonder what it means when it's not there.

If you want to add `enabled: true` properties to the others for consistency I would approve that change.

Reply:
@ckerr Done.